presto-common/src/main/java/com/facebook/presto/common/Page.java

```diff
@@ -418,12 +422,17 @@ public Page dropColumn(int channelIndex)
 
     private long updateRetainedSize()
```

Comment:
Might be worthwhile to add a dedicated test case additionally in `TestPage`?

Reply:
Gets tricky to handcraft. Let me try doing it. Another option is to use serde flow, but then it will be similar to existing test